docker/swe.Dockerfile

```diff
@@ -2,9 +2,12 @@ FROM ubuntu:jammy
 
 ARG TARGETARCH
 
+# Replay Dev Stuff
+EXPOSE 8080
+
 # Install third party tools
 RUN apt-get update && \
-    apt-get install -y bash gcc git jq wget g++ make && \
+    apt-get install -y bash gcc git jq wget g++ gfortran make && \
```

Comment:
@toshok - Do we need this for every docker image? feels like this is slowing things down quite a bit?